cfme/tests/automate/test_class.py

```diff
@@ -43,10 +43,11 @@ def namespace(request, domain):
 def test_class_crud(namespace):
     """
     Polarion:
-        assignee: dmisharo
+        assignee: ghubale
         casecomponent: Automate
         caseimportance: critical
         initialEstimate: 1/30h
+        tags: automate
```

Comment:
Why are we adding a tag that matches our component, just with different capitalization? This does not seem useful. The field in Polarion is free text, and we aren't using it for reporting.

Reply:
It will help me to sort my test cases as per my focus areas.
Yes, there are other ways to do it but it seems to be fast way to sort out tests.